#! /bin/sh -e
## 03_inadyn8ManPage.dpatch by  <shaul@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Documnet the acceptable Signals.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch)   patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@

--- inadyn.orig/man/inadyn.8	Thu Sep  8 03:50:08 2005
+++ man/inadyn.8	Sat Oct 15 10:51:31 2005
@@ -213,10 +213,8 @@
 .B inadyn
 will print some error messages. Those are harmless and should be 
 followed by 'OK' messages after the connection is back up.
-
 .SH Signals
 Any of the SIG_HUP, SIG_INT, SIG_QUIT will cause inadyn to terminate gracefully.
-
 .SH "SEE ALSO"
 .SS "Other manual pages"
 The syntax of the optional configuration file is given by
